projects/igniteui-angular/src/lib/drop-down/drop-down-item.base.ts

```diff
@@ -135,8 +140,15 @@ export abstract class IgxDropDownItemBase implements DoCheck {
             return;
         }
         this._selected = value;
+        this.selectedChange.emit(this._selected);
```

Comment:
@MayaKirova This code doesn't work, because drop-down-item overrides the setter. As a result when the selection is changed from the UI the model is not updated.